CMakeLists.txt

```diff
@@ -509,6 +509,10 @@ if(NOT SNMALLOC_HEADER_ONLY_LIBRARY)
   function(compile name TYPE ${ARGN})
     add_library(${name} ${TYPE} ${ARGN})
     target_link_libraries(${name} PRIVATE snmalloc)
+
+    # PRIVATE linking still touches INTERFACE level linking for unknown reasons.
```

Comment:
Could you expand this comment to include the link you mention in the PR:
https://cmake.org/cmake/help/latest/command/target_link_libraries.html#libraries-for-both-a-target-and-its-dependents

Then it is easier for someone (future me) to find out what you are referring.